From a94b1997f2ceb73ca7700268d0e0281704096d84 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 18 Feb 2019 18:04:53 +0000
Subject: [PATCH] TRITON-1017 iPXE should be provided directly from joyent/ipxe
 Reviewed by: Robert Mustacchi <rm@joyent.com> Approved by: Tim Foster
 <tim.foster@joyent.com>

---
 README.md | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index ec6f6e2..a070ddc 100644
--- a/README.md
+++ b/README.md
@@ -5,11 +5,15 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 # sdcboot
 
+*This repository is now retired, as we deliver (only) iPXE straight from the
+[ipxe](https://github.com/joyent/ipxe) repository. The below is of historical
+interest only.*
+
 This repository is part of the Joyent SmartDataCenter project (SDC).  For
 contribution guidelines, issues, and general documentation, visit the main
 [SDC](http://github.com/joyent/sdc) project page.
-- 
2.21.0

